pages/learn/develop/blocks.md

```diff
@@ -0,0 +1,128 @@
+
+## Creating your own block
+
+Since blocks are container images, the process to create them is very similar to creating any other container. To learn more about what makes a good candidate for a block, refer to this [blog post](https://www.balena.io/blog/how-to-make-your-own-balenablocks-simple-drop-in-edge-app-functionality/). 
+
+In order to create your own block. The code for your block should be in a GitHub project. It has to [include a balena.yml file](https://www.balena.io/docs/learn/deploy/deploy-with-balena-button/#balenayml-configuration-file) and a logo. Here's a balena.yml file of the [browser block](https://github.com/balenablocks/browser/) for reference:
+
+
+```yml
+name: browser
+description: >-
+  a hardware accelerated web browser to present internal and external URLs on a
+  connected display
+version: 2.3.2
+type: sw.block
+assets:
+  repository:
+    type: blob.asset
+    data:
+      url: 'https://github.com/balenablocks/browser'
+  logo:
+    type: blob.asset
+    data:
+      url: 'https://raw.githubusercontent.com/balenablocks/browser/master/logo.png'
+data:
+  defaultDeviceType: raspberrypi3
+  supportedDeviceTypes:
+    - raspberrypi4-64
+    - fincm3
+    - raspberrypi3
+    - raspberrypi3-64
+    - raspberrypi400-64
+    - intel-nuc
+    - genericx86-64-ext
+```
+
+Next, navigate to the `Blocks` tab in the sidebar and click the `Create block` button in the balenaCloud dashboard.
+
+<img src="/img/blocks/image.png" width="100%">
+
+Fill the details for your block in the modal, and click `Create new block` button to create your block.
+
+<img src="/img/blocks/create-block.png" width="70%">
+
+Next, we have to [push a new release][balena-push] from your project to your newly created block on balenaCloud. If you are new to this, then follow the [Getting Started][getting-started] guide. After the release has been created, your block is ready. You can manage, modify and iterate further on your block from the balenaCloud dashboard.
+
+## Release management for blocks
+
+To deploy new releases of your block, you can use the [balena-push][balena-push] command using the [balenaCLI](https://github.com/balena-io/balena-cli). The `Releases` tab in the sidebar will show the list of past releases of your block as well as the release that is currently selected as default.
+
+<img src="/img/blocks/release.png" width="80%">
+
+By default, the block set to `track latest`, which means that new releases are immediately tagged as the default release for the block. Expanding this _Default release_ dropdown menu displays all releases for the block, and you can select a specific release to pin the block to. When a particular release is selected, that release will be tagged as the default (or latest) release of your block. In future, releases will not be deployed until the _Default release_ is updated to a newer release or set back to `track latest`.
```

Comment:
By default, the block is set to*

Reply:
Expanding this _Default release_ dropdown menu displays all successful releases for the block*

Reply:
> default (or latest)

I don't think the `(or latest)` part is true or that we should use that phrasing 😓

Reply:
In the future, releases will*